### ISSUE TITLE
ReferenceError: Nn is not defined

### ISSUE THREAD

**Issue by hezhisheng1930** [darkreader/darkreader#12568]
### Prerequisites

- [X] I [searched for any existing report](https://github.com/darkreader/darkreader/issues?q=is%3Aissue) about this bug to avoid opening a duplicate.
- [X] I can reproduce this bug in a new, unmodified web browser profile with Dark Reader installed as the only extension.
- [X] I understand I need to use the [Broken Website Report template](https://github.com/darkreader/darkreader/issues/new?assignees=&labels=Broken+Website&template=broken-website-report.yml&title=%5BBroken+Website%5D+Replace+with+title) if this bug I am reporting occurs on a single website.

### Bug Description

I integrated darkreader into my website, but when I switched to night mode during the deployment, I got this error

### Website Address

https://www.hezhisheng.fun

### Steps To Reproduce

1. Switch to night mode in the upper right corner of the website
2. Open the console and get this error
![E85BBA9B-8C31-4a3e-870C-08CDA4FDDD3E](https://github.com/darkreader/darkreader/assets/63486587/b092b102-928f-484a-8646-3798d9fdca8e)

### Expected Behavior

Hope to stop this bug

### Actual Behavior

error

### Screenshots

![7e120a5959fee7d54d1f8482edb7fdd0](https://github.com/darkreader/darkreader/assets/63486587/8ab111b5-51ef-45d4-9b43-7263c3b6361e)

### Operating System

windows x64

### Web Browser name and version

google 125.0.6396.3

### Dark Reader version

4.9.81

**Comment by alexanderby** [comment]
Dark Reader's script is transpiled into ES2020. If you are transpiling further into ES5, the following line `{...oldDescriptor}` is transpiled into `Object.assign({}, oldDescriptor)`. `Nn` in your script is supposed to be this `Object.assign`, but it isn't there for some reason. Check your build configuration.

**Comment by hezhisheng1930** [comment]
> Dark Reader 的脚本被转译为 ES2020。如果您要进一步转译为 ES5，以下行将`{...oldDescriptor}`转译为`Object.assign({}, oldDescriptor)`.`Nn`在你的脚本中应该是 this `Object.assign`，但由于某种原因它不存在。检查您的构建配置。

是的，感谢你的建议，我修改了兼容配置，现在没有出现这个错误了

**Comment by hezhisheng1930** [comment]
> Dark Reader 的脚本被转译为 ES2020。如果您要进一步转译为 ES5，以下行将`{...oldDescriptor}`转译为`Object.assign({}, oldDescriptor)`。`Nn`在您的脚本中应该是这样`Object.assign`，但由于某种原因不存在。请检查您的构建的配置。

Yes, thank you for your suggestion. I have modified the compatibility configuration and this error has not occurred now
